src/components/user/tour/tour.js

```diff
@@ -1,6 +1,24 @@
 import React from "react";
-import Typography from "@material-ui/core/Typography";
+import { DataGrid } from '@material-ui/data-grid';
+import ToutApi from "../../../api/tour-client";
+import * as Const from './utils/constants';
```

Comment:
Do not import everything from the file. Import only constants that you need to use here.